website/content/docs/auth/gcp.mdx

```diff
@@ -88,9 +85,13 @@ management tool.
 
    If you are using instance credentials or want to specify credentials via
    an environment variable, you can skip this step. To learn more, see the
-   [Google Cloud Authentication](#authentication) section below.
+   [Google Cloud Credentials](#gcp-credentials) section below.
 
-1. Create a named role:
+   -> **Note**: If you're using a [Private Google Access](https://cloud.google.com/vpc/docs/configure-private-google-access)
+   environment, you will additionally need to configure your environment’s custom endpoints
+   via the [custom_endpoint](/api-docs/auth/gcp#custom_endpoint) configuration parameter.
+
+4. Create a named role:
```

Comment:
Looks like the numbering got messed up here (goes 1, 2, 4). I know it doesn't actually affect the way the page renders since the values themselves don't matter though, but might be worth changing all three to be `1.` for simplicity.

Reply:
Thanks! Ya I finally turned off the auto-incrementing setting in my editor after this :)